src/reynir_correct/errtokenizer.py

```diff
@@ -2208,13 +2273,16 @@ def is_valid_suggestion(
                         # about this new one, we yield no annotation
                         yield token
                     else:
-                        if Settings.DEBUG:
-                            print(
-                                "Suggested '{1}' instead of '{0}'".format(
-                                    token.txt, corrected_txt
+                        rtok = token
+                        if "W001" not in ignore_rules:
+                            if Settings.DEBUG:
```

Comment:
This DEBUG output can probably be deleted now.

Reply:
Done